src/components/ChallengeModal/ChallengeModal.tsx

```diff
@@ -143,7 +143,7 @@ export class ChallengeModal extends Modal<ChallengeModalProperties, any> {
             conf: {
                 mode: this.props.mode,
                 username: "",
-                bot_id: data.get("challenge.bot", 0),
```

Comment:
I suspect this was an accident given the recent pattern removal of the `get("...", 0)` - or was this on purpose?